app/namespace.go

```diff
@@ -1490,6 +1552,42 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				return c.failoverNamespace(ctx)
 			},
 		},
+		{
+			Name:    "update",
```

Comment:
Just `update` is confusing. Sounds like it updates all the namespace configuration. 
I would suggest calling it `update-disaster-recovery` in step with the other namespace commands.

Reply:
this is what I originally did. I changed it based on feedback.